hydra/redisgraph_demo1.py

```diff
@@ -0,0 +1,274 @@
+    # put and get method for possible status in class possible_status:
+
+    possibleStatus(status_node)
+
+
+def supported_property(supportedprop_node):
+
+    title = 'obj_supportedclasses_supportedProperty_title'
+    # put and get method for supp_prop title
+    node = Node(
+        label="node",
+        alias="property_title",
+        properties={
+            "prop": title})
+    redis_graph.add_node(node)
+    create_graph(supportedprop_node, node, "has_title")
+
+    properties = 'obj_supportedclasses_supportedProperty_property'
+    # put and get method for properties class Property
+    property_links = "all links of property"
+    property_node = Node(
+        label="node",
+        alias="property",
+        properties={
+            "prop": property_links})
+    redis_graph.add_node(property_node)
+    create_graph(node, property_node, "has_property")
+
+    Property(property_node)
+
+    required = 'obj_supportedclasses_supportedProperty_requires'
+    # put and get method for reuires
+    node2 = Node(label="node", alias="requires", properties={"prop": required})
+    redis_graph.add_node(node2)
+    create_graph(node, node2, "has_requires")
+
+    readonly = 'obj_supportedclasses_supportedProperty_readonly'
+    # put and get method for readonly
+    node3 = Node(
+        label="node",
+        alias="readonly",
+        properties={
+            "prop": str(readonly)})
+    redis_graph.add_node(node3)
+    create_graph(node, node3, "is_readonly")
+
+    writeonly = 'obj_supportedclasses_supportedProperty_writeonly'
+    # put and get method for writeonly
+    node3 = Node(
+        label="node",
+        alias="writeonly",
+        properties={
+            "prop": str(writeonly)})
+    redis_graph.add_node(node3)
+    create_graph(node, node3, "is_writeonly")
+
+
+def possibleStatus(status_node):
+
+    status_code="obj_supportedclasses_supportedOperation_possiblestatus_statusCode"
+    # put and get method
+    node = Node(
+        label="node",
+        alias="status_code",
+        properties={
+            "prop": str(status_code)})
+    redis_graph.add_node(node)
+    create_graph(status_node, node, "has_status")
+
+
+def Property(property_node):
+
+    link = 'obj_supportedclasses_supportedProperty_property_Links'
+    # put and get method for links
+    node = Node(label="node", alias="property_link", properties={"prop": link})
+    redis_graph.add_node(node)
+    create_graph(property_node, node, "has_links")
+
+    supportedoperation = 'obj_supportedclasses_supportedProperty_property_supportedoperation'
+    node1 = Node(
+        label="node",
+        alias="supportedoperation",
+        properties={
+            "prop": supportedoperation})
+    redis_graph.add_node(node1)
+    create_graph(node, node1, "has_supportedoperation")
+
+
+def create_graph(source_node, dest_node, predicate):
+    edge = Edge(source_node, predicate, dest_node)
+    redis_graph.add_edge(edge)
+
+
+apidocumentation()
+
```

Comment:
Hey sandeep, instead of just storing the whole documentation in Redis graph. Try to visualize things first. For example first you should think about a rough graph structure that can be easily used to store the complete api structure along with what kind of data needs to be stored at each endpoint. Forget the APIDoc, try to visualize this first for a simple API you know. Then try to find a way to extract all that required information from APIDOC and after that you can retrieve data from the server and add it to the graph for querying. Makes sense?